src/include/kdump/uifunctions.rb

```diff
@@ -1515,6 +1517,12 @@ def InitDumpFormat(_key)
         UI.ChangeWidget(Id("DumpFormat"), :Value, "compressed_format")
       elsif Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT") == "lzo"
         UI.ChangeWidget(Id("DumpFormat"), :Value, "lzo_format")
+      elsif Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT") == "snappy"
+        UI.ChangeWidget(Id("DumpFormat"), :Value, "snappy_format")
+      elsif Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT") == "zstd"
+        UI.ChangeWidget(Id("DumpFormat"), :Value, "zstd_format")
+      elsif Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT") == "raw"
+        UI.ChangeWidget(Id("DumpFormat"), :Value, "raw_format")
```

Comment:
just nitpick, but this code is quite hairy already. So if you want to use it as oppurtinity to learn a bit ruby there are two options.

1. easier one is to use `case` statement instead of bunch of elsifs so something like

```ruby
case Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT")
when "ELF"
  UI.ChangeWidget(Id("DumpFormat"), :Value, "elf_format")
when "compressed"
  UI.ChangeWidget(Id("DumpFormat"), :Value, "compressed_format")
...
```

2. bigger change, but smarter is to have mapping hash and use it in this method.

this can look like:

```ruby
FORMAT_MAPPING = {
  "ELF" => "elf_format",
  "compressed" => "compressed_format",
  ...
}

def InitDumpFormat(_key)
  value = FORMAT_MAPPING[Ops.get(Kdump.KDUMP_SETTINGS, "KDUMP_DUMPFORMAT")] || "none_format"
  UI.ChangeWidget(Id("DumpFormat"), :Value, value)
```